lib/mal_import.rb

```diff
@@ -102,7 +102,7 @@ def metadata
         ja_jp: begin @sidebar.css('div:contains("Japanese:")')[0].text.gsub("Japanese: ", "") rescue nil end
       }.compact,
       synopsis: begin
-        synopsis = @main_noko.css('td td:contains("Synopsis")')[0].text.gsub("EditSynopsis", '').split("googletag.cmd.push")[0]
+        synopsis = @main_noko.css('td td:contains("Synopsis")')[0].text.gsub("EditSynopsis", '').split("EditBackground")[0].split("googletag.cmd.push")[0]
```

Comment:
Left it in for backwards compatibility, In case they decide to move the background information into it's own table column.